lib/rspec/core/configuration.rb

```diff
@@ -1531,12 +1528,12 @@ def self.delegate_to_ordering_manager(*methods)
         end
       end
 
-      # @macro delegate_to_ordering_manager
+      # @!method seed=(value)
       #
       # Sets the seed value and sets the default global ordering to random.
       delegate_to_ordering_manager :seed=
 
-      # @macro delegate_to_ordering_manager
+      # @!method seed
```

Comment:
This is doc'ing as a `void` return type.  Do we need a `@return` to doc it properly?

Reply:
yes. I'm going to leave that out of this PR for now - there are many methods like this that are doc'ed as `void`.

Reply:
https://github.com/rspec/rspec-core/issues/2494 for follow on